lib/raidenclient/RaidenClient.ts

```diff
@@ -72,6 +72,7 @@ type Channel = OpenChannelPayload & {
 class RaidenClient extends BaseClient {
   private port: number;
   private host: string;
+  public address?: string;
```

Comment:
I always moved the declaration of public variables above the private ones. Is that something we should care about?

Reply:
Yeah I think that's a good idea, I'd be in favor of following that convention. There's a tslint rule for it too that we can add: https://palantir.github.io/tslint/rules/member-ordering/